[BUG]Cannot load ogbg-molfreesolv

**Describe the bug**
When using `glb.dataloading.get_glb_graph(ogbg-molfreesolv)`,  ogbg-molfreesolv fails
**To Reproduce**
simply run `glb.dataloading.get_glb_graph(ogbg-molfreesolv)`


**Screenshots**

```
dataset = 'ogbg-molfreesolv'

    def test_data_loading(dataset):
        """Test data loading for a given dataset.

        Test if get_glb_graph, get_glb_task, and get_glb_dataset
        can be applied successfully.
        """
        directory = os.getcwd() + "/datasets/" + dataset
        task_list = []
        for file in os.listdir(directory):
            if fnmatch.fnmatch(file, "task*.json"):
                with open(directory + "/" + file,  encoding="utf-8") as f:
                    task_dict = json.load(f)
                    if task_dict["type"] not in SUPPORTED_TASK_TYPES:
                        f.close()
                        return
                print(os.path.splitext(file)[0])
                task_list.append(os.path.splitext(file)[0])
        try:
>           _ = glb.dataloading.get_glb_graph(dataset)

tests/test_data_loading.py:27:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
glb/dataloading.py:75: in get_glb_graph
    return read_glb_graph(metadata_path, device=device, verbose=verbose)
glb/graph.py:44: in read_glb_graph
    return _get_multi_graph(data, device)
glb/graph.py:94: in _get_multi_graph
    g: dgl.DGLGraph = _get_single_graph(data)
glb/graph.py:82: in _get_single_graph
    g = _get_homograph(data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

data = {'Edge': {'EdgeFeature': tensor([[0, 0, 0],
        [0, 0, 0],
        [0, 0, 0],
        ...,
        [0, 0, 0],
    ...,
        ...,
        [5, 0, 4,  ..., 2, 0, 1],
        [5, 0, 4,  ..., 2, 0, 1],
        [7, 0, 2,  ..., 2, 0, 1]])}}

    def _get_homograph(data):
        """Get a homogeneous graph from data."""
        edges = data["Edge"].pop("_Edge")  # (num_edges, 2)
        src_nodes, dst_nodes = edges.T[0], edges.T[1]
>       num_nodes = data["Graph"]["_NodeList"].shape[-1]
E       KeyError: '_NodeList'

glb/graph.py:130: KeyError
```

--- a/glb/graph.py
+++ b/glb/graph.py
@@ -86,12 +86,12 @@ def _get_single_graph(data, device="cpu", hetero=False):
 
 def _get_multi_graph(data, device="cpu"):
     """Initialize and return a list of Graph instance given data."""
+    # Extract the whole graph
+    g: dgl.DGLGraph = _get_single_graph(data)
+
     node_list = data["Graph"].pop("_NodeList")
     edge_list = data["Graph"].pop("_EdgeList", None)
     graphs = []
-
-    # Extract the whole graph
-    g: dgl.DGLGraph = _get_single_graph(data)
 
     # Check array type before assigning
     for attr, array in data["Node"].items():

